cli/index.ts

```diff
@@ -527,7 +526,10 @@ function createYargsCli(cli: ICli) {
       command.format,
       command.description,
       yargsChainer => createOptionsChain(yargsChainer, command),
-      command.processFn
+      async (argv: any) => {
```

Comment:
ew, `any`

Reply:
I've given this a slightly better type, but there isn't much we can do here, it's really a map/dict mapping from argument names to arguments (which could be anything)